apps/calendar/locales/calendar.en-US.properties

```diff
@@ -44,7 +44,7 @@ advanced-settings-add-account=Add Account
 field-user.placeholder=Username
 field-password.placeholder=Password
 field-full-url.placeholder=URL
-modify-account-setting-up=Please wait while your account is set up
+modify-account-setting-up=Setting up account
```

Comment:
But it looks like it's only used in  only this section . How can I find it whether it's used it or not in calendar code?